modules/concepts/commands.md

```diff
@@ -77,6 +77,66 @@ services:
 
 The command should be now available using `./bin/console your-module:export`.
 
+
+### Autoloading
+
+The instructions above are assuming the module is installed via composer. In case you follow the same procedure but wish to add the command in your already existing module in `modules/<module_directory>` then the procedure requires an extra step. 
+
+For example with a structure like this 
+
+```tree
+modules/module_directory/
+├── config
+│   └── services.yml
+├── config.xml
+├── module_directory.php
+├── src
+│   └── Command
+│       └── SyncCommand.php
+```
+
+You would also need to namespace your src directory in `docroot/composer.json`
```

Comment:
I'm not sure to fully understand what docroot is

Reply:
docroot is a term from apache to refer to the document root where the website resides. It's the directory where each website resides in classic web apps.

Reply:
We might need to use another word if we want this documentation to be as easy as possible for all type of readers ;)